P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+    <p>The TEI Abstract Model can be expressed as a hierarchy of attribute-value matrices (AVMs)
+      of various types and of various levels of complexity, nested or grouped in various ways. At
+      the most abstract level, an AVM consists of an information container and the value
+      ('contents') of that container.</p>
+    <p>A simple example of an XML serialization of such structures is, on the one hand, the opening
+      and closing tags that delimit and name the container, and, on the other, the content enclosed
+      by the two tags that constitues the value. An analogous example is an
+      attribute name and the value of that attribute.</p>
+    <p>In a TEI XML example of two equivalent serializations expressing the name-value pair
+      <hi style="font-family:monospace">&lt;part-of-speech,common-noun></hi>, namely 
+      <hi style="font-family:monospace">&lt;pos>commonNoun&lt;/pos></hi> and 
+      <hi style="font-family:monospace">pos="common-noun"</hi>, one would
+      classify the element <gi>pos</gi> and the attribute <att>pos</att> as containers (mapping onto
+      the first member of the relevant name-value pair), while the <hi style="font-family:monospace">text()</hi>
+      content of <gi>pos</gi> or the string/symbolic value of <att>pos</att> would be seen as
+      mapping onto the second member of the pair.</p>
+    <p>The <ident type="class">att.datcat</ident> class provides means of addressing the containers
+      and their values, while at the same time providing a way to interpret them in the context of
+      external taxonomies or ontologies. Aligning e.g. both the <gi>pos</gi> element and the
+        <att>pos</att> attribute with the same value of an external reference point (i.e., an entry
+      in an agreed taxonomy) affirms the identity of the concept serialised by both the element container
+      and the attribute container, and optionally provides a definition of that concept (in the case at hand,
+      the concept <soCalled>part of speech</soCalled>).</p>
+    <p>The value of the <ident type="class">att.datcat</ident> attributes should be a PID
+      (persistent identifier) that points to a specific — and, ideally, shared — taxonomy or
+      ontology. Among the resources that to a smaller or greater extent can be used as inventories
+      of (more or less) standardized linguistic categories, are the GOLD ontology, <ref
+        target="https://www.clarin.eu/content/clarin-concept-registry">CLARIN CCR</ref>, <ref
+        target="https://github.com/acoli-repo/olia">OLiA</ref> or 
+      <ref target="https://datcatinfo.net/">TermWeb's DatCatInfo</ref>, and also the 
+      <ref target="https://universaldependencies.org/">Universal
+      Dependencies</ref> inventory, on the assumption that its URI are going to persist. Apart from these
+      global resources, it is imaginable to address a project-specific taxonomy as well, if global
+      interoperability is not the goal.</p>  
```

Comment:
Subjunctive. (Or is it conditional?) Either way, I am wondering if “were” is more correct than “is”.

A project-specific taxonomy might be useful for other reasons, too, no? Most notably if a project wanted a taxonomy different from those globally available, typically because more or less precision than provided by the globally available ones is desired.

I am also quite confused by the “as well” and the “global interoperability not the goal”. Do you mean “as well” as “we can imagine a project specific taxonomy in addition to our imagining of global taxonomies” or “we can imagine a TEI user pointing to both a project specific taxonomy and a global taxonomy”? I suspect the former, but either way this ambiguity is problematic.

Thus perhaps “It is imaginable that a project may choose to address a local taxonomy instead, but this risks losing the enormous advantage of global interchangeability.” or some such.

Reply:
Super, thanks.

Reply:
(and I meant a simple conditional here)